### ISSUE TITLE
Portal Feedback: Josh

### ISSUE THREAD

**Issue by 10thfloor** [onflow/developer-portal#724]
**Portal Overview Instructions:**

**Developer Portal Overview:**
Give a more thorough overview. I want to know what the main repo is for the developer portal, the high level overview of how it generates docs in its own repo and how it pulls docs from other repos.
Also, it would be good to know how to get docs to show up in the main navigation.

**Learn**
It says it should not contain learning content related to specific tools, but also says it should contain tutorials, guides, and conceptual documentation. Usually those things are related to specific tools, so it isn’t clear to me where the distinction is

**Tools**
Seems like this should only be formal specs, or is it also tutorials?

**Content Integration Options:**
It says once content is merged into the main branch, it is available, but does that mean that the main docs repo needs to be updated to include that repo first, or is it just automatic?
Does the branch have to be called main, or can it be called master?

**Flow-docs.json:**
The linked schema is not clear and not very easy to read. It would be nice to have that in a better format
It would be great to also have the cadence example annotated with explanations of what each block is doing, with screenshots of how that translates to its actual appearance on the docs site
Each section of the flow-docs.json section just needs some more context of how it uses that information to actually show stuff on the site

**Content URLS:**
What is subsection?

**SEO:**
What default is provided

**Page Content**
This looks like it is kind of restating some of the stuff in the flow.json section. It would probably be better to find a way to combine those sections so you’re not having to scroll back and forth
Might also be good to define header hero
Define href for those not familiar
The images and links sections are in the same section here as headers and menus, but they don’t seem like they are related

**Content Previews:**
The preview banner screenshot image is not displaying properly

**Contributions Guidelines Instructions:**
What do I do if I want to integrate a repo into an existing section of the site? For example, I have the cadence style guide repo that I would like to integrate into the Cadence section of the docs site without having to put it in the actual Cadence repo
If this doc is public, which it should be, you’ll need to change the slack [#flow-documentation](https://dapperlabs.slack.com/archives/C01E563SA92) channel to a public discord channel
